src/UI/Component/ViewControl/Factory.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\ViewControl;
+/**
+ * This is how the factory for UI elements looks.
+ */
```

Comment:
I'm sure this exists in many other locations, but I just noticed that this might not be very informative.